cpe-remap.yaml

```diff
@@ -29,18 +33,34 @@ mappings:
     vendor: cisco
     products:
       adaptive_security_appliance: adaptive_security_appliance_software
+      apic: application_policy_infrastructure_controller
       pix: pix_firewall_software
       telepresence: telepresence_video_communication_server_software
+  crushftp:
+    products:
+      crushftp_web_interface: crushftp
+  data_domain:
+    vendor: dell
```

Comment:
EMC bought Data Domain in 2009.  Dell bought EMC in 2016. There are multiple data domain CPEs under EMC and Dell but none under Data Domain itself.

 For example, these are BOTH valid NIST issued CPEs and BOTH have *different* CVEs against them.

cpe:2.3:o:emc:data_domain:5.6
cpe:2.3:o:dell:emc_data_domain_os:5.6

I chose the most modern version with the expectation that it would be the most likely to work in the future and if anything was deprecated it would be the EMC ones. 

I'll reach out to NIST and see what they say.

NIST CPE search - https://nvd.nist.gov/products/cpe/search/results?namingFormat=2.3&keyword=data+domain

Reply:
Ok, the NIST CPE team responded. They are deprecating the EMC CPEs in favor of the Dell version (`cpe:2.3:o:dell:emc data_domain_os`). This is in line with the change in this PR.